README.md

```diff
@@ -15,41 +15,26 @@ Install
 -------------
 1. Create a conda or virtual environment and activate it
 
-2. Update pip and wheel in your environment:
+2. We recommens that you update pip and wheel in your environment:
   ```
   pip install -U pip wheel
   ```
-3. Install [l2logger](https://github.com/darpa-l2m/l2logger).
-   If you have ssh keys configured for GitHub, install like so:
+3. Install tella
    ```
-   pip install git+ssh://git@github.com/darpa-l2m/l2logger.git
-   ```
-   If this does not work, try:
-   ```
-   pip install git+https://github.com/darpa-l2m/l2logger.git
-   ```
-   Otherwise, clone the l2logger repository and install:
-   ```
-   git clone https://github.com/darpa-l2m/l2logger
-   pip install ./l2logger
-   ```
-4. Clone this repository:
-   ```
-   git clone git@github.com:darpa-l2m/tella.git
-   ```
-   or
-   ```
-   git clone https://github.com/darpa-l2m/tella.git
-   ```
-5. Install the tella package and its dependencies:
-   ```
-   pip install "./tella[minigrid]"
+   pip install tella
```

Comment:
minigrid is still optional on our install. Do we want to add it with atari and dev below?

Reply:
I changed minigrid to be required. It's small enough that it seemed fine to include by default.